content/index.mdx

```diff
@@ -1,19 +1,20 @@
 ---
-template: "base"
+template: 'base'
 ---
-import Header from "components/organisms/SiteHeader"
-import Footer from "components/organisms/SiteFooter"
-import SubscribeSection from "components/molecules/SubscribeSection"
 
-import GitFlowIcon from "media/icons/gitflow.svg"
-import ReportsIcon from "media/icons/reports.svg"
-import BoxIcon from "media/icons/cube.svg"
+import Header from 'components/organisms/SiteHeader'
+import Footer from 'components/organisms/SiteFooter'
+import SubscribeSection from 'components/molecules/SubscribeSection'
```

Comment:
where do all the changes in this file come from?

Reply:
> _All the double/single quote changes have been made by the Git hooks, so I guess they're fine._

I'm baffled by the eccentricities of the commit hooks, but it became like that somehow after running them. 🤔  I wonder if I ran them manually and missed the configuration file.

Reply:
same with #81 btw

Reply:
these changed don't look like a formatter/linter lol

Reply:
Please refer to https://github.com/iterative/cml.dev/pull/81#discussion_r690673965